styles/styles.css

```diff
@@ -1010,14 +1012,23 @@ main .section.our-history .year.highlight .icon-calendar {
 }
 
 /* Divider as Section Metadata */
-.section hr.divider {
+.section hr.divider.primary {
   border: 0;
   height: 4px;
   background-color: var(--mt-global-horizontal-line-color);
   width: 50%;
   margin: calc(26px + 10px + 3%) auto auto;
 }
 
+.section hr.divider.secondary {
+  border: 0;
+  height: 4px;
+  background-color: var(--mt-global-horizontal-line-color-blue);
+  width: 34px;
+  margin: calc(26px + 10px + 3%) auto auto;
+}
```

Comment:
CSS properties which are the same for primary and secondary could be consolidated into a common selector:

```
.section hr.divider {
  border: 0;
  height: 4px;
  margin: calc(26px + 10px + 3%) auto auto;
}
```